run-tests-with-build-kogito-apps-parent/kogito-apps-project-sources-test/pom.xml

```diff
@@ -20,7 +20,8 @@
       container.image.infinispan,
       container.image.keycloak,
       container.image.kafka,
-      container.image.mongodb
+      container.image.mongodb,
+      quarkus.container-image.build
```

Comment:
I noticed that the same set of properties is used for all tests of the same repository, e.g. there are same properties defined for runtimes, apps, examples. Shouldn't we put them to parents to remove duplicates?

Reply:
as part of BAQE-2455 I specifically didn't want to move those into parent and stated as `The parameters are not intended to be moved into parent poms, to keep execution related config in a single pom.`

I was first inclined to do so, but then it would also make sense to move the plugin definition into parent (cause that's duplicated too) .. and then the lowest-level poms would be imo harder to read, but anyway, if desirable would do as a follow-up task.